sdlf-ddk-lightweight/python/data_lake/pipelines/sdlf_base_stack.py

```diff
@@ -79,13 +80,17 @@ def __init__(
 
         dataset_names: set[str] = set()
         pipelines: Dict[str, SDLFPipeline] = {}
+        
         # loop through values in parameters.json and create the necessary resources for each pipeline
         for customer_config in customer_configs:
             dataset = customer_config["dataset"]
             team = customer_config["team"]
             pipeline_type = customer_config.get(
                 "pipeline", StandardPipeline.PIPELINE_TYPE
             )
+            orchestration = customer_config.get(
```

Comment:
Maybe leave this as default behavior if not in config

Reply:
Hi, I can make that change. Could you please confirm if i should set the orchestration default in the SDLFHeavyTransformConfig / SDLFLightTransformConfig or leave it in the base_stack ?